### DISCUSSION TITLE
Citations?

### DISCUSSION THREAD

**Discussion by Paganiniana** [browserengineering/book#903] [General]
Hey, all! 

I'm wondering if there are any citations for [this](https://browser.engineering/graphics.html#:~:text=discrete%20action%20in-,100ms,-%2D%20below%20that%20threshold) comment in Chapter 2 of the book. I'd love to dig into the research on this (or at least read the abstracts). 

Thanks in advance!

**Comment by chrishtr** [comment]
Good point; I'll add a citation to the book. In the meantime, here is a link:

https://www.nngroup.com/articles/response-times-3-important-limits/

This data was also the inspiration for the numbers in the "RAIL" model:

https://web.dev/rail/

**Reply by Paganiniana** [reply]
Great stuff, thanks!
